packages/ai-chat-ui/src/browser/chat-input-widget.tsx

```diff
@@ -288,8 +292,8 @@ const ChatInput: React.FunctionComponent<ChatInputProperties> = (props: ChatInpu
                     <span
                         className="codicon codicon-send option"
                         title="Send (Enter)"
-                        onClick={!props.isEnabled ? undefined : () => submit(editorRef.current?.document.textEditorModel.getValue() || '')}
-                        style={{ cursor: !props.isEnabled ? 'default' : 'pointer', opacity: !props.isEnabled ? 0.5 : 1 }}
+                        onClick={!props.isEnabled || isMessageEmpty ? undefined : () => submit(editorRef.current?.document.textEditorModel.getValue() || '')}
+                        style={{ cursor: !props.isEnabled || isMessageEmpty ? 'default' : 'pointer', opacity: !props.isEnabled || isMessageEmpty ? 0.5 : 1 }}
```

Comment:
Nit: It might be a better to rather add a CSS class if it is disabled and assign the pointer and opacity in the CSS file if this CSS class is assigned. This way it is more easy to customize and doesn't pollute the code here so much.

Reply:
Good point, programmatic CSS is a nightmare to override

Reply:
Not introduced by this PR, though :-)
I had a look at https://github.com/eclipse-theia/theia/pull/14750
We should merge https://github.com/eclipse-theia/theia/pull/14750 first and then fix it again, as the structure completly changes there.